src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageRestoreBar.xaml.cs

```diff
@@ -158,6 +161,14 @@ public virtual async Task<bool> GetMissingAssetsFileStatusAsync(string projectId
             if (nuGetProject?.ProjectStyle == ProjectModel.ProjectStyle.PackageReference &&
                 nuGetProject is BuildIntegratedNuGetProject buildIntegratedNuGetProject)
             {
+                // If there are no packages we dont need the assets file in the PM UI
```

Comment:
```suggestion
                // If there are no packages, we don't need the assets file in the PM UI
```

Reply:
Can you also very briefly explain in the comment why this is the case? One sentence should be enough.